docs/crio.8.md

```diff
@@ -170,6 +171,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--default-ulimits**="": Ulimits to apply to containers by default (name=soft:hard) (default: []) (default: [])
 
+**--drop-infra-ctr**: determines whether we drop the infra container when appropriate. Requires ManageNSLifecycle to be true (default: true)
```

Comment:
it would be nice to have a bit more detail to what "when appropriate" means.

Reply:
fixed!